src/main/java/uk/gov/hmcts/divorce/common/event/CreateTestCase.java

```diff
@@ -96,8 +94,8 @@ public AboutToStartOrSubmitResponse<CaseData, State> midEvent(
 
         final CaseData data = details.getData();
         try {
-            UUID uuid = UUID.fromString(data.getCaseInvite().applicant2UserId());
-            log.info("{}", uuid);
+//            UUID uuid = UUID.fromString(data.getCaseInvite().applicant2UserId());
+//            log.info("{}", uuid);
```

Comment:
It would be helpful if we add a comment on why we commented these lines of code.